Locals/zhTW.lua

```diff
@@ -0,0 +1,54 @@
+addonName, CraftSim = ...
+
+CraftSim.LOCAL_TW = {}
+
+function CraftSim.LOCAL_TW:GetData()
+    return {
+        -- REQUIRED:
+        [CraftSim.CONST.TEXT.STAT_INSPIRATION] = "靈感",
+        [CraftSim.CONST.TEXT.STAT_MULTICRAFT] = "複數製造",
+        [CraftSim.CONST.TEXT.STAT_RESOURCEFULNESS] = "精明",
+        [CraftSim.CONST.TEXT.STAT_CRAFTINGSPEED] = "製造速度",
+        [CraftSim.CONST.TEXT.EQUIP_MATCH_STRING] = "裝備：",
+        [CraftSim.CONST.TEXT.ENCHANTED_MATCH_STRING] = "附魔：",
+        [CraftSim.CONST.TEXT.INSPIRATIONBONUS_SKILL_ITEM_MATCH_STRING] = "在製作期間獲得靈感時，提高所提供的的技能",
```

Comment:
I do not really know what the translation is here, but it should be a part of the sentence describing the % bonus to the inspiration bonusskill that certain profession tools give, like the blacksmithing epic hammer

![image](https://user-images.githubusercontent.com/9341090/211251849-ef0bbf22-f687-487d-8244-04b15710279a.png)

Reply:
![image](https://user-images.githubusercontent.com/21115928/211309269-629f1af4-8b82-416e-9ac8-0024f7cf9819.png)
Yes I have matched the string according to the hammer.